root/static/scripts/guess-case/utils.js

```diff
@@ -173,7 +173,8 @@ export function titleString(gc, is, forceCaps) {
     // we got an 'til (Until = 'til), lowercase.
     // we got an 'way (Away = 'way), lowercase.
     // we got an 'round (Around = 'round), lowercased
-    } else if (isApostrophe(gc.i.getPreviousWord()) && lc.match(/^(s|round|em|ve|ll|d|cha|re|til|way|all)$/i)) {
+    // we got a 'mon (Come on = C'mon), lowercase
```

Comment:
Should we uppercase the C below, too (in the "Make it O'Titled, Y'All" part)? (If so an extra small test for that would be nice.)

Reply:
Good idea. Just changed the new test a bit for both.